HISTORY.rst

```diff
@@ -1,6 +1,45 @@
 =======
 History
 =======
+3.41.0 (2012-08-04)
+-------------------
+
+Changes
+
+* CCI has a new Metadata ETL task that allows adding fields to existing layouts. Sections can be selected using an index (#2766)
```

Comment:
It has other ways of targeting where to put the field too. I don't think it makes sense to call out section indexes specifically.

Also release notes for tasks should always mention the name of the task config!

```suggestion
* We added a new Metadata ETL task, `add_page_layout_fields`, that allows adding fields to existing layouts. (#2766)
```